src/adapter/index.ts

```diff
@@ -172,7 +176,7 @@ export function getFieldPaths(
                 // In V2, it's stored and typed like every other timestamp.
                 // We force conversion from ISO 8601 string => epoch millis
                 // string by overriding the type from string to timestamp.
```

Comment:
comment is out-of-date

Reply:
I think it's accurate? This is pointing out an inconsistency in the v1 schema where refreshedAt is serialized as a timestamp, but is typed as a string. If we left the type as a string instead of overriding it, then we wouldn't convert the ISO timestamp to an epoch millis string.

Reply:
I think I was being too literal.  The override type is now `epoch_millis_string`.

Reply:
Ah. True.